modules/platform/nuxeo-template-rendering/nuxeo-template-rendering-core/src/test/java/org/nuxeo/template/TestTemplateRenderingService.java

```diff
@@ -274,4 +279,26 @@ protected TemplateSourceDocument createTemplateSourceDoc(String templateString,
 
         return templateSource.getAdapter(TemplateSourceDocument.class);
     }
+
+    @Test
+    public void testFormatDate() {
+        String defaultFormat = "MM/dd/yyyy";
+        String format1 = "yyyy MMM dd HH:mm:ss";
+        String format2 = "EEEE dd MMMMM yyyy";
+        Locale locale = Locale.getDefault();
+
+        GregorianCalendar calendar = new GregorianCalendar(2013, 11, 31);
+        Date dt = calendar.getTime();
+
+        SimpleDateFormat default_sdf = new SimpleDateFormat(defaultFormat, locale);
```

Comment:
Really sorry but Kevin's point was that in Java we don't use `foo_bar` for variable names, so it also applies to `default_sdf`...